dataactbroker/handlers/fileHandler.py

```diff
@@ -1343,6 +1344,10 @@ def list_submissions(page, limit, certified, sort='modified', order='desc'):
     """ List submission based on current page and amount to display. If provided, filter based on
     certification status """
     sess = GlobalDB.db().session
+    db = GlobalDB.db()
+    metadata = MetaData(bind=db.engine)
+
+    submission_updated_view = Table('submission_updated_at_view', metadata, autoload=True, info=dict(is_view=True))
```

Comment:
Unfortunately no, that was part of the reason I had to figure this nonsense out. The problem is that, with the other ones, we basically create an object that automatically reflects the table and organizes the columns nicely when we create a model in the appropriate file. We can't make a model for a view, we have to just create it manually in the alembic migration, which means there's no quick-access method of doing it. We could turn this whole process into a class that we can call (which would still involve all this exact same stuff but in a different file/location), but I figured if it's just being done once for now we probably don't need to go that deep.